Resources/Prototypes/Entities/Objects/Consumable/Food/skewer.yml

```diff
@@ -157,3 +157,25 @@
       - state: skewer-corn2
       - state: skewer-mushroom2
       - state: skewer-tomato1
+
+- type: entity
+  name: snake kebab
+  parent: FoodSkewerBase
+  id: FoodMeatSnakeKebab
+  description: Snake meat on a stick. It's a little tough.
+  components:
+  - type: Sprite
+    layers:
+      - state: skewer
+      - state: skewer-snake1
+      - state: skewer-snake2
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 10
+        - ReagentId: Oculine
+          Quantity: 2
+        - ReagentId: Tricordrazine
+          Quantity: 2
```

Comment:
why does it have a bunch of healing meds in it?

Reply:
Oculine is like the effect of the cobra having the blindness toxin, intended that way, but maybe there will be more snakes... Yeah, I'll remove the medicine. There's always the thought of giving some kind of feature in my head.